Hybrid-state.md

```diff
@@ -0,0 +1,51 @@
+## Overview
+
+Oppia currently uses **Protractor** for end-to-end testing but as the Angular team plans to end the development of Protractor at the end of 2022, we are plannning to migrate all these tests to **WebdriverIO**.
+Till the time the migration will be complete the tests will be present in **Hybrid State**, i.e the tests will be present in both webdriverIO and protractor.
```

Comment:
If you want this to be a separate paragraph, you'll need to add a blank line above it. Ditto elsewhere

Reply:
Done.